helix-view/src/editor.rs

```diff
@@ -174,6 +174,8 @@ pub struct Config {
     pub indent_guides: IndentGuidesConfig,
     /// Whether to color modes with different colors. Defaults to `false`.
     pub color_modes: bool,
+    /// Whether the cursor matches the mode as specified by `ui.cursor.insert` and `ui.cursor.select`. Defaults to `false`.
```

Comment:
```suggestion
    /// Whether to theme the cursor according to the editor mode. Defaults to `false`.
```

Reply:
We might want to clarify the docs for `color_modes` too: "Whether to theme the mode statusline element according to the editor mode. Defaults to `false`"